x-pack/plugins/cases/public/components/user_action_tree/user_action_content_toolbar.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { UserActionCopyLink } from './user_action_copy_link';
+import { UserActionPropertyActions } from './user_action_property_actions';
+
+interface UserActionContentToolbarProps {
+  id: string;
+  getCaseDetailHrefWithCommentID: (commentID: string) => string;
```

Comment:
Please make `ID` => `Id`, `getCaseDetailHrefWithCommentId`. Matches up with other uses of `Id`